### ISSUE TITLE
enhance(datasplitter): Explore different ways to split the data

### ISSUE THREAD

**Issue by DarkMenacer** [DarkMenacer/Machine-Learning-Pipelines#4]
- Look into validation set and epochs
- Look into other ways of data splitting other than random split
- Check inbuilt sklearn datasplit function as well
